File: problem_solving/staircase/answer.java
static void staircase(int n) {
    for (int i = 1; i <= n; i++) {
        String hash = "";
        for (int j = n; j > 0; j--) {   
            if (j<=i) {
                hash += "#";
            } else {
                hash += " ";
            }
        }
        System.out.println(hash);
    }
}

File: problem_solving/staircase/answer.js
function staircase(n) {
    for (let i = 1; i <= n; i++) {
        let hash = "";
        for (let j = n; j > 0; j--) {   
            if (j<=i) {
                hash += "#";
            } else {
                hash += " ";
            }
        }
        console.log(hash);
    }
}
